keras_nlp/models/bart/bart_tokenizer.py

```diff
@@ -92,16 +94,26 @@ def __init__(
         merges,
         **kwargs,
     ):
+        # Special tokens.
+        start_token = "<s>"
+        pad_token = "<pad>"
+        end_token = "</s>"
+
+        if "unsplittable_tokens" in kwargs:
```

Comment:
I agree that this is probably better UX, but just in terms of our own code maintenance, I think we can ditch this. If users pass this argument they will get an error message abut duplicate kwargs, and can check the code here. It is fairly self explanatory.